### ISSUE TITLE
Unlabelled buttons

### ISSUE THREAD

**Issue by kieranlittle667** [Villarrealized/sonos-web#145]
Hi,
I am blind and rely on a screenreader. Currently the buttons in the now-playing section at the bottom of the page are not labelled, so it is unclear to me what they do, and also the mute button is labelled as "prepend icon".
Apart from no accessible obvious way to group rooms, everything else seems completely accessible.
Thanks for the great work, love the controller.
